Core/MIPS/ARM/ArmAsm.cpp

```diff
@@ -15,6 +15,8 @@
 // Official git repository and contact information can be found at
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
+#include "ppsspp_config.h"
+#if PPSSPP_ARCH(ARM)
```

Comment:
One pattern that we should remove is conditionally adding members to the structs. That would remove some preprocessor defines and make it simpler and then easier to mock for tests.